specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2020-04-01-preview/EventGrid.json

```diff
@@ -5896,9 +5896,32 @@
             "modelAsString": true
           }
         },
+        "partnerTopicReadinessState": {
+          "description": "The readiness state of the corresponding partner topic.",
+          "enum": [
+            "NotActivatedByUserYet",
```

Comment:
>NotActivatedByUserYet [](start = 13, length = 21)

Do you need Yet at the end?

Reply:
yes.. We need it as separate case of Deactivated as we have separate behavior for this case than the case where partner topic is activated and then deactivated later.